### ISSUE TITLE
Warn about Cesium ion demo token

### ISSUE THREAD

**Issue by kring** [CesiumGS/cesium-unreal#1040]
When CesiumJS is using its default (and frequently deactivated) Cesium ion token, it displays a message like this:

> This application is using Cesium's default ion access token. Please assign Cesium.Ion.defaultAccessToken with an access token from your ion account before making any Cesium API calls. You can sign up for a free ion account at https://cesium.com/

Unlike CesiumJS, Cesium for Unreal does not include a default Cesium ion token. However, the Cesium for Unreal _Samples_ project does, so perhaps we should have a similar warning in the Samples project.

We should be clear about what we're trying to accomplish, though:

1. Displaying such a message may encourage people to try out Cesium ion, which is great.
2. If a user copies the token from the Samples into their own project, no logic in the Samples project itself could possibly show this message, because it won't exist in the user's new project. So should Cesium for Unreal _itself_ detect tokens from the samples account and display the message? I think this will require an extra request and therefore extra latency before showing a tileset from ion, which isn't great. Perhaps this is an unlikely scenario anyway.
3. If a user starts with the Samples project and then modifies it to make it their own, it's definitely a good thing to show a message so that the user will be aware of the token before it is deactivated and breaks their project. Do we think people commonly do this?

In short, the situation in Cesium for Unreal is quite a bit different from CesiumJS, and so we should be clear about what we're trying to accomplish and how we want to accomplish it before we implement anything. CC @shehzan10 

Once this is settled here, we should apply the same solution in Cesium for Unity and Cesium for O3DE.
